libs/components/filter-bar/testing/src/modules/filter-bar/filter-item-harness.ts

```diff
@@ -44,16 +44,16 @@ export class SkyFilterItemHarness extends SkyComponentHarness {
   }
 
   /**
-   * Gets the filter item name.
+   * Gets the filter item value.
    */
-  public async getName(): Promise<string> {
-    return await (await this.#getFilterName()).text();
+  public async getFilterValue(): Promise<string | undefined> {
+    return await (await this.#getFilterValue())?.text();
   }
 
   /**
-   * Checks if the filter item has an active value.
+   * Gets the filter item label text.
```

Comment:
```suggestion
   * Gets the filter item label.
```
I don't _think_ we need "text" here, but please ignore this suggestion if I'm overlooking some distinction.